index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link rel="stylesheet" href="./style.css">
+      <title>Document</title>
+      <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,700;1,300;1,600&display=swap"
+            rel="stylesheet">
+</head>
+
+<body>
+      <header>
+            <nav class="nav">
+                  <p>My logo</p>
+                  <img src="./image/Menu.png" alt="">
+            </nav>
+      </header>
+      <section class="heading-section">
+            <h1 class="name">I'm Blossom Glad to see you!</h1>
+            <p class="description">I’m a software developer! I can help you  build a product , feature or website
+                  Look through some of my work and experience! If you like what you see and  have a
+                  project you need coded, don’t hestiate to contact me.</p>
+            <p class="connect">LET’S CONNECT</p>
+            <ul class="icons">
+                  <li>
+                        <a href="#"><img src="./image/twitter.png" alt="ttwitter icon"></a>
+                  </li>
+                  <li>
+                        <a href="#"><img src="./image/Linkdin.png" alt="Linkedin icon" /></a>
+                  </li>
+                  <li>
+                        <a href="#"><img src="./image/meduim.png" alt="medium icon" /></a>
+                  </li>
+                  <li>
+                        <a href="#"><img src="./image/git.png" alt="git icon" /></a>
+                  </li>
+                  <li>
+                        <a href="#"><img src="./image/angelist.png" alt="angelist icon" /></a>
+                  </li>
```

Comment:
- [ **OPTIONAL** ] I think it would be better to add your correct links on each social media(if you have its account account) to make your portfolio more professional